homeworks/vlad.zelenko_undercover-agent/5-rxjs/js/common.js

```diff
@@ -0,0 +1,18 @@
+const { fromEvent } = window.rxjs;
+const { map, pairwise, filter } = window.rxjs.operators;
+const headerElement = document.querySelector('[data-header]');
+const scrollEvent = fromEvent(window, 'scroll');
+const step = 50;
+
+scrollEvent.pipe(
+    map(() => window.pageYOffset),
+    pairwise(),
+    filter(([before, after]) => Math.abs((after - before) > step || (before - after) > step)),
```

Comment:
You didn't get my point 😄 
Here what I meant:

```js
filter(([before, after]) => Math.abs(before - after) > step),
```

The code is simple and clear: calculate the pixel difference, got the module of the number and check is it higher than the step.